addon/components/code-inline.js

```diff
@@ -1,3 +1,5 @@
+/* global Prism */
```

Comment:
No problem! I was searching for a usable syntax highlighting ember addon. I almost had to build my own from scratch if I haven't found Prism and this awesome addon =)